source/quick-start.txt

```diff
@@ -28,74 +28,85 @@ instance.
 Set up Your Project
 -------------------
 
-Install Node and NPM
+Install Node and npm
 ~~~~~~~~~~~~~~~~~~~~
 
 Ensure your system has Node.js version 10 or later and a compatible
```

Comment:
I decided to refer the user to check the compatibility table since this page is used across all versions of the documentation. I think doing this would also be easier to maintain moving forward.